docs/data/material/varnish/logos/ProjectsLogosDemo.tsx

```diff
@@ -44,28 +44,63 @@ export default function Demo() {
       <Item>
         <Center>
           <ImageListItemBar title="Prior" position="top" />
-          <logos.Prior />{' '}
+          <logos.Prior />
         </Center>
       </Item>
 
       <Item>
         <Center>
           <ImageListItemBar title="Semantic Scholar" position="top" />
-          <logos.SemanticScholar />{' '}
+          <logos.SemanticScholar />
         </Center>
       </Item>
 
       <Item>
         <Center>
           <ImageListItemBar title="Fairness" position="top" />
-          <logos.Fairness />{' '}
+          <logos.Fairness />
         </Center>
       </Item>
 
       <Item>
         <Center>
           <ImageListItemBar title="Incubator" position="top" />
-          <logos.Incubator />{' '}
+          <logos.Incubator />
+        </Center>
+      </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="EarthRanger" position="top" />
+          <logos.EarthRanger className="earthranger_logo" />
+        </Center>
+      </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="Skylight" position="top" />
+          <logos.Skylight className="skylight_logo" />
+        </Center>
+      </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="Climate" position="top" />
+          <logos.Climate />
+        </Center>
+      </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="Wildlands" position="top" />
+          <logos.Wildlands />
```

Comment:
er...
mono should default to true...
so that our old logos make sense as coded

so maybe we alter it to 'polychrome'
(all existing ones are mono)